include/mbedtls/ssl.h

```diff
@@ -1011,7 +1011,56 @@ typedef void mbedtls_ssl_export_keys_t( void *p_expkey,
  */
 struct mbedtls_ssl_config
 {
-    /* Group items by size (largest first) to minimize padding overhead */
+    /* Group items by size and reorder them to maximize usage of immediate offset access.    */
+
+    /*
+     * Flags (bitfields)
+     */
+
+    unsigned int MBEDTLS_PRIVATE(endpoint) : 1;      /*!< 0: client, 1: server               */
```

Comment:
On arm at least, the size of a bit-field container with the type `unsigned int` is 4 bytes (or larger if there are more than 32 bits). (See [AAPCS](https://developer.arm.com/documentation/ihi0042/j/?lang=en#aapcs32-section7-1-7)). So if we're sorting by size, this should count as 4, not as &lt1.

Reply:
Yep, you're absolutely right! 
I had to misunderstood some piece of [documentation](https://developer.arm.com/documentation/dui0491/i/C-and-C---Implementation-Details/Structures--unions--enumerations--and-bitfields?lang=en).
Ok, I will fix it!

Reply:
Done.